extensions/amp-subscriptions/amp-subscriptions.md

```diff
@@ -262,97 +483,215 @@ The vendor service configuration must reference the service ID and can contain a
+```html
+<script type="application/ld+json">
+  {
+    "@context": "http://schema.org",
+    "@type": "NewsArticle",
+    "isAccessibleForFree": false,
+    "publisher": {
+      "@type": "Organization",
+      "name": "The Norcal Tribune"
+    },
+    "hasPart": {...},
+    "isPartOf": {
+      "@type": ["CreativeWork", "Product"],
+      "name" : "The Norcal Tribune",
+      "productID": "norcal_tribune.com:basic"
+    }
+  }
+</script>
+```
 
-Pingback is optional. It's only enabled when the "pingbackUrl" property is specified.
+Thus, notice that:
 
-By default, as the body, pingback POST request receives the entitlement object returned by the "winning" authorization endpoint. However if the config for the "local" service contains `pingbackAllEntitlements: true` the body will contain an array of all the entitlments received, from all services, including those which do not grant access.
+1.  The product ID is "norcal_tribune.com:basic" (`"productID": "norcal_tribune.com:basic"`).
+2.  This document is currently locked (`"isAccessibleForFree": false`).
 
-**Important:** The pingback JSON object is sent with `Content-type: text/plain`. This is intentional as it removes the need for a CORS preflight check.
+#### Microdata markup
 
-## Actions
+Using Microdata, the markup could look like this:
+
+```html
+<div itemscope itemtype="http://schema.org/NewsArticle">
+  <meta itemprop="isAccessibleForFree" content="false" />
+  <div
+    itemprop="isPartOf"
+    itemscope
+    itemtype="http://schema.org/CreativeWork http://schema.org/Product">
+    <meta itemprop="name" content="The Norcal Tribune" />
+    <meta itemprop="productID" content="norcal_tribute.com:basic" />
+  </div>
+</div>
+```
+
+A usable configuration will provide `NewsArticle` typed item with `isAccessibleForFree` property and a subitem of type `Product` that specifies the `productID`.
+
+In this example:
+
+1.  The product ID is "norcal_tribune.com:basic" (`"productID": "norcal_tribune.com:basic"`).
+2.  This document is currently locked (`"isAccessibleForFree": false`).
+
+The configuration is resolved as soon as `productID` and `isAccessibleForFree` are found. It is, therefore, advised to place the configuration as high up in the DOM tree as possible.
 
-Actions are provided in the "local" service configuration in the "actions" property. It's a named set of action. Any number of actions can be configured this way, but two actions are required: "login" and "subscribe".
 
-All actions work the same way: the popup window is opened for the specified URL. The page opened in the popup window can perform the target action, such as login/subscribe/etc, and it's expected to return by redirecting to the URL specified by the "return" query parameter.
+## Attributes
+### `subscription-action`
+
+An action declared in the "actions" configuration can be marked up using `subscriptions-action` attribute.
```

Comment:
Why would I want to mark it up? Whats the value of this attribute?

Reply:
Added some more text here to describe what the actions do.